src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <div class="dashboard-board-list">
+        <p-field-title class="p-field-title">
+            <template>
+                <span>{{ fieldTitle }}</span>
+                <span class="board-count">({{ dashboardListByBoardSets.length }})</span>
+            </template>
+        </p-field-title>
+        <p-board :board-sets="dashboardListByBoardSets"
+                 class="board"
+        >
+            <template #item-content="{board}">
+                <div class="board-item-title-wrapper">
+                    <div class="favorite-button-wrapper">
+                        <favorite-button :item-id="board.domain_dashboard_id"
+                                         :favorite-type="FAVORITE_TYPE.DASHBOARD"
+                                         scale="0.666"
+                        />
+                    </div>
+                    <span class="board-item-title">{{ board.name }}</span>
+                </div>
+                <p class="board-item-description">
+                    What item I fill in this area?
+                </p>
+                <p-label :class="{'viewers-label': true, 'private-label': board.viewers === 'PRIVATE'}"
+                         :text="board.viewers === 'PUBLIC' ? 'Public' : 'Private'"
+                         :left-icon="board.viewers === 'PUBLIC' ? 'ic_public' : 'ic_private'"
+                />
+                <p-label v-for="(label, idx) in board.labels"
+                         :key="`${board.domain_dashboard_id}-label-${idx}`"
+                         :text="label"
+                         clickable
+                />
+            </template>
+        </p-board>
+    </div>
+</template>
+
+<script lang="ts">
+import { computed, reactive, toRefs } from 'vue';
+
+import { PBoard, PFieldTitle, PLabel } from '@spaceone/design-system';
+
+import { FAVORITE_TYPE } from '@/store/modules/favorite/type';
+
+import FavoriteButton from '@/common/modules/favorites/favorite-button/FavoriteButton.vue';
+
+export default {
+    name: 'DashboardBoardList',
+    components: {
+        PLabel, FavoriteButton, PBoard, PFieldTitle,
+    },
+    props: {
+        fieldTitle: {
+            type: String,
+            default: undefined,
+        },
+        dashboardList: {
+            type: Array,
+            default: () => [],
+        },
+    },
+    setup(props) {
+        const state = reactive({
+            boardItemIconButtonSets: [
+                {
+                    iconName: 'ic_edit',
+                    eventAction: () => console.log('edit!'),
+                },
+                {
+                    iconName: 'ic_duplicate',
+                    eventAction: () => console.log('dup!'),
+                },
+                {
+                    iconName: 'ic_trashcan',
+                    eventAction: () => console.log('delete!'),
+                },
+            ],
```

Comment:
question: Is it updatable state?

If it isn't, I suggest not to put this constant to reactive object. 
If it is mutable state, just ignore this comment :)

Reply:
It may refactor soon!